projects/managed-hubs/support.md

```diff
@@ -93,6 +93,9 @@ We should try to respond to all support-related communications within one workin
 - Give the user updates every day or two as we work to resolve the issue.
 - When it is resolved, tell the user what you’ve done to resolve the issue.
 - Close any FreshDesk tickets and support issues that are related to this request.
+- In some cases, it might be useful to also write down a private note in the FreshDesk
+  ticket with a short summary of the situation and/or add links to relevant discussions
+  from other platforms.
```

Comment:
```suggestion
- In some cases, it might be useful to also write down [a private note in the FreshDesk ticket](https://support.freshdesk.com/support/solutions/articles/37580-private-notes-for-internal-sharing)
  with a short summary of the situation and/or add links to relevant discussions
  from other platforms.
```